src/magento-payments/payouts.md

```diff
@@ -13,42 +13,36 @@ You can download payout transactions in a .csv file format for use in existing a
 
 ## Availability
 
-On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payouts** to see payout transactions.
+On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payouts**.
 
 ![Magento Admin - Payouts]({% link images/images-ee/system-reports.png %}){: .zoom}
 _Payout transactions in the Admin_
 
-From the Payouts view, you can customize the timeframe of the payouts you want to view by selecting specific dates or viewing transactions for pre-set periods, such as _Last 30 days_ or _Year to date_.
-
 ## View transactions
 
 By default, 30 days of transactions are shown in the grid.
 
 Scroll to the left and right to view [information for each payout transaction](#column-descriptions) in the daily report, including transaction date, reference ID, invoice number, and payment method details.
 
-The number of payout transactions that are visible, for the default 30 day timeframe or a [customized timeframe](#customize-transactions-timeframe), are shown above the payouts grid as a records count.
-
 ### Customize transactions timeframe
 
-If you want to see all payout transactions for a time period other than the default last 30 days, you can customize the payout transaction timeframe for specified dates.
+From the Payouts view, you can customize the timeframe for the payout transactions you want to view by entering a specific timeframe or selecting a date range from the date picker:
 
 1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payouts**.
-1. Click the Payout transaction dates calendar selector filter.
-1. Select the desired timeframe for your report---_Yesterday_, _Last 7 days_, _Last 30 days_, _Last 90 days_, _Year to date_, or _Custom range_. See the [Default transactions timeframes section](#default-transactions-timeframes) for more information.
-   - If you selected _Custom range_, choose the applicable date range.
-1. View the payout transactions for your specified dates in the grid.
+1. Enter in a date range, such as `2021-01-02 - 2012-01-12`, in a YEAR-MONTH-DAY format, or select a specific date range from the date picker.
```

Comment:
I am not sure if YYYY-MM-DD is a universal format that most people would understand in place of YEAR-MONTH-DAY

Reply:
Good point. I'll check into that and modify.